src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Startup.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Altinn.Platform.Authorization
+{
+    /// <summary>
+    /// Authorization startup.
+    /// </summary>
+    public class Startup
+    {
+        /// <summary>
+        ///  Initializes a new instance of the <see cref="Startup"/> class
+        /// </summary>
+        /// <param name="configuration">The configuration for the authorization component</param>
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        /// <summary>
+        /// Gets authorization project configuration.
+        /// </summary>
+        public IConfiguration Configuration { get; }
+
+        /// <summary>
+        /// Configure authorization setttings for the service.
+        /// </summary>
+        /// <param name="services">the service configuration.</param>
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.Configure<CookiePolicyOptions>(options =>
```

Comment:
not needed and can be removed

Reply:
also should add these lines instead:
services.AddMvc().AddControllersAsServices();
services.AddSingleton(Configuration);

Reply:
Done